lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1347,9 +1426,26 @@ class MongoClientInterface {
      * @param {Logger}log logger instance
      * @param {Function} cb callback containing error
      * and BulkWriteResult
+     * @param {object} [params] request params
      * @return {undefined}
      */
-    internalDeleteObject(collection, bucketName, key, filter, log, cb) {
+    internalDeleteObject(collection, bucketName, key, filter, log, cb, params = null) {
```

Comment:
should keep `cb` as last parameter

Reply:
Any reason why we cannot put optional params after the callback as we do in other functions? Like the `originOp`, `isRetry` and all?

Reply:
Done in https://github.com/scality/Arsenal/pull/2134/files#diff-9c207befbd8f4d79a3b6e3d7d143444e804ca1684236b8fd9fa0f9e03c53477aR1433